spec/unit/recipes/boxcutter_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'spec_helper'
+
+describe 'macos::boxcutter' do
+  context 'macOS High Sierra 10.13' do
```

Comment:
These don't seem to pass for me:

```
1) macos::boxcutter macOS High Sierra 10.13 behaves like a successful convergence does not raise an exception
     Failure/Error: expect { chef_run }.to_not raise_error

       expected no Exception, got #<Chef::Exceptions::ValidationFailed: Property value must be one of: TrueClass, FalseClass, String, Integer, Float!  You passed [nil].> with backtrace:
         # /var/folders/x5/6v905wsn4t5_crp4twz0d0w80000gp/T/d20180226-70017-xetk8p/cookbooks/macos/recipes/boxcutter.rb:29:in `block in from_file'
         # /var/folders/x5/6v905wsn4t5_crp4twz0d0w80000gp/T/d20180226-70017-xetk8p/cookbooks/macos/recipes/boxcutter.rb:27:in `from_file'
         # ./spec/spec_helper.rb:28:in `block (2 levels) in <top (required)>'
         # ./spec/spec_helper.rb:33:in `block (4 levels) in <top (required)>'
         # ./spec/spec_helper.rb:33:in `block (3 levels) in <top (required)>'
     Shared Example Group: "a successful convergence" called from ./spec/unit/recipes/boxcutter_spec.rb:8
     # ./spec/spec_helper.rb:33:in `block (3 levels) in <top (required)>'
```
Looks like `node['macos']['uid']` is `nil` - how is this set?